baselines/webworker.generated.d.ts

```diff
@@ -2737,29 +2708,6 @@ declare var ReadableStream: {
     new<R = any>(underlyingSource?: UnderlyingSource<R>, strategy?: QueuingStrategy<R>): ReadableStream<R>;
 };
 
-interface ReadableStreamBYOBReader {
```

Comment:
@MattiasBuelens, could you review the Streams part of this removal? Thanks!

Reply:
`UnderlyingByteSource`, `ReadableByteStreamControllerCallback` and [this `ReadableStream` constructor overload](https://github.com/microsoft/TSJS-lib-generator/blob/0b6caa7b6a1add40024da034863ae8f6fa6a2bf6/baselines/dom.generated.d.ts#L12345) should be removed as well.

You might want to have a look at 361a87076aef97268105f9b8f85c686d701df69e (part of #890).

Reply:
Hang on, where did `WritableStream` and `TransformStream` go? There are still types referencing them (like `GenericTransformStream` and `ReadableStream.pipeTo`).

(I understand why they would need to be removed, but it still makes me sad to see them go... 😞)

Reply:
The bright side is that this patch only disables some of your work but not removes them all. It can be enabled again when browsers implement them 👍

Reply:
BTW, WritableStream and TransformStream are still alive as there is at least a single implementation and thus not covered here. This patch is intentionally only removes totally unimplemented ones to reduce the diff size.

Reply:
Streams changes LGTM. 👍 Happy to see `WritableStream` and `TransformStream` are still alive! 😁

I think it's best to wait with landing #890 until after this PR lands. I'll see if I can rebase my PR to not *completely* remove the readable byte stream definitions, and instead rely on your work to auto-remove them. 🙂